File: output/gemma4-e4b/avatar/Python/Java/atcoder_ABC164_A.java
import java.util.Scanner;

public class atcoder_ABC164_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        if (scanner.hasNextLine()) {
            String line = scanner.nextLine();
            String[] parts = line.split(" ");
            if (parts.length == 2) {
                try {
                    int S = Integer.parseInt(parts[0]);
                    int W = Integer.parseInt(parts[1]);

                    if (S <= W) {
                        System.out.println("unsafe");
                    } else {
                        System.out.println("safe");
                    }
                } catch (NumberFormatException e) {
                    // Handle case where input parts are not valid integers
                }
            }
        }
        scanner.close();
    }
}

File: output/gemma4-e4b/avatar/Java/Python/atcoder_ABC164_A.py
import sys

def solve():
    # Read all input lines and take the first one, mimicking the Java logic
    input_data = sys.stdin.read().strip()
    
    if not input_data:
        return

    try:
        # Split the first line by spaces
        numlist = input_data.split()
        
        # Assuming exactly two numbers are provided
        sheep = int(numlist[0])
        wolve = int(numlist[1])
        
        if wolve >= sheep:
            print("unsafe")
        else:
            print("safe")
    except IndexError:
        # Handle case where not enough numbers are provided
        pass
    except ValueError:
        # Handle case where conversion to int fails
        pass

solve()